modules/system/lang/en/lang.php

```diff
@@ -449,6 +449,10 @@
     ],
     'media' => [
         'invalid_path' => "Invalid file path specified: ':path'.",
+        'folder_size_items' => 'item(s)',
+        'metadata_image_width' => 'Width',
+        'metadata_image_height' => 'Height',
+        'metadata_video_duration' => 'Duration',
```

Comment:
metadata should be an array with subitems under it so that keys are `metadata.image_width` instead of `metadata_image_width` - this will reduce the unnecessary duplication of the `metadata_` prefix.